solr/core/src/test/org/apache/solr/search/TestCpuAllowedLimit.java

```diff
@@ -185,7 +185,9 @@ public void testDistribLimit() throws Exception {
                 "stages",
                 "prepare,process",
                 "cpuAllowed",
-                "50"));
+                "50",
+                "multiThreaded",
+                "false"));
```

Comment:
#2590 proposes to explicitly not (yet) support multi-threading when query limits are used, and with that then the test change here could be reverted.